TESTING_WITH_IAF-TEST.md

```diff
@@ -53,3 +53,5 @@ Press [ Start ], sit back, relax, do some stretches, and let's hope for the best
 ### Troubleshooting
 
 * Some parts of the iaf-test module rely on proprietary modules. To tell Maven that it should download these modules, go to Window > Preferences > Maven > User Settings. If you already have a _settings.xml_ file, press the "Open file" link. Otherwise, browse to _C:/Users/(your name)/.m2/_ and create a _settings.xml_ file. Edit the file by adding your own repository or the [frankframework nexus repository](https://nexus.frankframework.org/content/groups/private/) as [mirror](https://maven.apache.org/guides/mini/guide-mirror-settings.html).
+* When your IP-address is dynamically generated, you may have problems connecting to your database that runs in a docker image. The Larva tests reference the database host by a DNS name, `host.docker.internal`. Docker may not automatically update the IP address to which this name refers when your computer is assigned a new IP address. To see whether you have this issue, do `ping host.docker.internal` in a command prompt. If you cannot reach this address, refresh your docker network. You can do that using docker desktop. Press the cog wheel to the top. In the left-hand menu, choose Resources | Network. Update the docker subnet.
```

Comment:
Cog wheel is geen Engels (of je moet het aan elkaar schrijven). Ik zou 'settings icon (cogwheel)' gebruiken.

Reply:
Verwerkt.